tasks/main.yml

```diff
@@ -101,10 +101,23 @@
   delegate_to: "{{ bastion_host | default(hyper) }}"
   when: guest_exists | failed
 
+- name: Clear known_hosts for new VM
```

Comment:
This is a good addition. Much nicer to have this done automatically instead of having old leftover host identities lurking about in known_hosts.